test/extended/util/test.go

```diff
@@ -401,8 +405,7 @@ var (
 			`\[Driver: iscsi\]`, // https://bugzilla.redhat.com/show_bug.cgi?id=1711627
 			`\[Driver: ceph\]\[Feature:Volumes\] \[Testpattern: Pre-provisioned PV \(default fs\)\] subPath should verify container cannot write to subpath`,
 
-			`\[Driver: aws\] \[Testpattern: Dynamic PV \(default fs\)\] provisioning should access volume from different nodes`, // https://bugzilla.redhat.com/show_bug.cgi?id=1711688
-			`\[Driver: nfs\] \[Testpattern: Dynamic PV \(default fs\)\] provisioning should access volume from different nodes`, // https://bugzilla.redhat.com/show_bug.cgi?id=1711688
+			`\[Testpattern: Dynamic PV \(default fs\)\] provisioning should access volume from different nodes`, // https://bugzilla.redhat.com/show_bug.cgi?id=1711688
```

Comment:
Does this lose valuable tests (i.e. are there now tests not being run because of this)?  Or were NFS and AWS the only ones that ran during the suite anyway and they were both sipped?

Reply:
It's a normal test like any other, it does not bring any great value.

Problem with the test on our clusters is that we run each node in a different AWS zone. The test needs at least two nodes in each zone and thus it fails.
We're trying to fix the test upstream: https://github.com/kubernetes/kubernetes/pull/78697
(approval would be nice).

Reply:
Wouldn't this pattern block NFS from being tested against vsphere clusters?

Reply:
Can you link to a bug to restore this test?  Someone later on should have an easy reference for why it is disabled.

Reply:
Well, there is a link to a bug.

> Wouldn't this pattern block NFS from being tested against vsphere clusters?

Yes, it would. It's IMO better to skip the test everywhere instead of ugly while-listing regexp (in a blacklist!). The test is being fixed and may be even merged before this PR lands, we're just missing approval.

Reply:
Now that #23361 is merged, no change in skipped tests is necessary.